scheduler-server/scheduling-dsl-compiler/src/libs/scheduler-ast.ts

```diff
@@ -23,6 +23,16 @@ export interface ActivityRecurrenceGoal {
   interval: number,
 }
 
+export interface ActivityExpression {
+  type: string
```

Comment:
We should codegen out the activity types so its enforced that we have correct values here

Reply:
We aren't generating any part of the AST currently, just the DSL. I think the AST should be static - in which case this field would remain a string.

Reply:
Good point, so I suppose its just in the fluent dsl where we need to be more specific